wbooks/src/app/screens/BookList/components/BookItem/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+import noBook from '@assets/img_book_placeholder.png';
+
+import { STYLES } from './styles';
+
+type Props = {
+  item: {
+    title: string;
+    author: string;
+    imageUrl: string | null;
+  };
```

Comment:
Extract `Book` interface => 
`interface Props { item: Book };`
```
interface Book {
    title: string;
    author: string;
    imageUrl: string | null;
}
```
You'll gonna need it